habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -373,3 +409,134 @@ def forward(self, observations, rnn_hidden_states, prev_actions, masks):
         x, rnn_hidden_states = self.state_encoder(x, rnn_hidden_states, masks)
 
         return x, rnn_hidden_states
+
+
+class ImageNavResNetNet(Net):
```

Comment:
Looks like a lot of code is reused form `PointNavResNetNet`. Maybe, worth to create common ancestor class.